### PR TITLE
[ProxyManagerBridge] remove deprecated features

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by xabbuh** [comment]
#22818 must be merged first to make tests pass

File: src/Symfony/Bridge/ProxyManager/composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": "^7.1.3",
-        "symfony/dependency-injection": "~3.4|~4.0",
+        "symfony/dependency-injection": "~4.0",
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/ProxyManager/composer.json]: can't we try harder to not add this?

Comment (xabbuh) [inline src/Symfony/Bridge/ProxyManager/composer.json]: In fact, we do not need to change anything here. Fixed.

**Comment by nicolas-grekas** [comment]
rebase unlocked

**Comment by xabbuh** [comment]
rebased

**Review by Simperfit — APPROVED** [review]

**Review by nicolas-grekas — APPROVED** [review]
:+1:

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**